tests/functional/backbeat/listLifecycleOrphanDeleteMarkers.js

```diff
@@ -0,0 +1,332 @@
+            }),
+        next => s3.deleteObject({ Bucket: bucketName, Key: keyName }, next),
+        next => s3.deleteObject({ Bucket: bucketName, Key: keyName, VersionId: versionId }, next),
+    ], cb);
+}
+
+runIfMongoV1('listLifecycleOrphanDeleteMarkers', () => {
+    let bucketUtil;
+    let s3;
+    let date;
+
+    before(done => {
+        bucketUtil = new BucketUtility('account1', { signatureVersion: 'v4' });
+        s3 = bucketUtil.s3;
+
+        return async.series([
+            next => s3.createBucket({ Bucket: testBucket }, next),
+            next => s3.createBucket({ Bucket: emptyBucket }, next),
+            next => s3.createBucket({ Bucket: nonVersionedBucket }, next),
+            next => s3.putBucketVersioning({
+                Bucket: testBucket,
+                VersioningConfiguration: { Status: 'Enabled' },
+            }, next),
+            next => s3.putBucketVersioning({
+                Bucket: emptyBucket,
+                VersioningConfiguration: { Status: 'Enabled' },
+            }, next),
+            next => async.times(3, (n, cb) => {
+                createOrhanDeleteMarker(s3, testBucket, `key${n}old`, cb);
+            }, next),
+            next => {
+                date = new Date(Date.now()).toISOString();
+                return async.times(5, (n, cb) => {
+                    createOrhanDeleteMarker(s3, testBucket, `key${n}`, cb);
+                }, next);
+            },
+        ], done);
+    });
+
+    after(done => async.series([
+        next => removeAllVersions({ Bucket: testBucket }, next),
+        next => s3.deleteBucket({ Bucket: testBucket }, next),
+        next => s3.deleteBucket({ Bucket: emptyBucket }, next),
+        next => s3.deleteBucket({ Bucket: nonVersionedBucket }, next),
+    ], done));
+
+    it('should return empty list of orphan delete markers if bucket is empty', done => {
+        makeBackbeatRequest({
+            method: 'GET',
+            bucket: emptyBucket,
+            queryObj: { 'list-type': 'orphan' },
+            authCredentials: credentials,
+        }, (err, response) => {
+            assert.ifError(err);
+            assert.strictEqual(response.statusCode, 200);
+            const data = JSON.parse(response.body);
+
+            assert.strictEqual(data.IsTruncated, false);
+            assert(!data.NextMarker);
+            assert.strictEqual(data.MaxKeys, 1000);
+            assert.strictEqual(data.Contents.length, 0);
+            return done();
+        });
+    });
+
+    it('should return empty list of orphan delete markers if prefix does not apply', done => {
+        makeBackbeatRequest({
+            method: 'GET',
+            bucket: testBucket,
+            queryObj: { 'list-type': 'orphan', prefix: 'unknown' },
+            authCredentials: credentials,
+        }, (err, response) => {
+            assert.ifError(err);
+            assert.strictEqual(response.statusCode, 200);
+            const data = JSON.parse(response.body);
+
```

Comment:
extra newline (or would make more sense to have this newline above https://github.com/scality/cloudserver/pull/5045/files#diff-d32bd21f47e01240317ed9a86946889e0d04888726d87d2d5d038252c6f6aaf8R118 otherwise)